index.html

```diff
@@ -95,12 +95,13 @@ <h2 class="about">About myself</h2>
       <div class="formsection">
         <h3 class="styleh3">CONTACT US</h3>
         <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-        <form action="" class="formstyle">
-          <input type="text" placeholder="Text input">
-          <input type="email" placeholder="email">
-          <textarea name="" id="" cols="30" rows="10" placeholder="Enter text here"></textarea>
+        <form action="https://formspree.io/f/moqrojze" method="POST" class="formstyle">
+          <input type="text" placeholder="Text input" maxlength="30" required>
+          <input type="email" placeholder="email" required>
+          <textarea name="text" cols="30" rows="10" placeholder="Enter text here" maxlength="500" required></textarea>
+          <button type="submit" class="button_pro"><span>Submit</span></button>
+
         </form>
```

Comment:
Figma Design          |  Your design    
:-------------------------:|:-------------------------:
![Template 4-main-mobile](https://user-images.githubusercontent.com/69638013/177376016-18ba8609-762f-436a-8366-08bb2ab08e25.png) | ![127 0 0 1_5501_index html](https://user-images.githubusercontent.com/69638013/177376131-917ab12d-7a74-4275-bfaa-8f9cac0637e5.png)


- [ ] Kindly center paragraph content and the form, see screenshot above 👍🏿  
- [ ] Please add button interactions such as active (when it's pressed), and remove button outline so your design looks exactly like the Figma template
- [ ] Kindly align the bottom social media icons to look like the Figma design 👍🏿

Reply:
Hello @prolajumokeoni  are u there? i relaly dont want to waste my trials, so this is good for you?
![image](https://user-images.githubusercontent.com/70304755/177381789-e8dbef74-074f-400f-8300-59c926f0a2a0.png)

Reply:
I have solved all your code reviewer comments, please l hope dont waste my free trials because l did all in the github instructions this is designing the previous project.s